<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Edit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Работа с данными</string>
    </property>
    <widget class="QPushButton" name="pushButton_AddNewEntry">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить новую запись</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ChangeEntry">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить запись</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DeleteEntry">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить запись</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_SaveLoad">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>430</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Сохранить/Загрузить</string>
    </property>
    <widget class="QPushButton" name="pushButton_SaveToFile">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить в файл</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LoadFromFile">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить из файла</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Данные</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_WorkplacePosition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Phone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_SaveLoad</zorder>
   <zorder>groupBox_Edit</zorder>
   <zorder>tableView</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
